packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -86,19 +95,42 @@ function FlowsheetDataGrid(props) {
     rowHeight,
     onCellSelect,
     onClearSelectedCells,
+    intl,
   } = props;
 
+  // Replace each non-header cell that contains no content with a dash indicating "No results".
+  const parsedRows = rows.map(
+    (row) => ({
+      ...row,
+      cells: row.cells.map(
+        (cell, index) => ((!cell.content && index !== 0) ? {
```

Comment:
Is the index really the "column"?  Are we excluding that column because it is the row header?  If so, it would probably be good to add a comment.

Reply:
Added a comment explaining this